test/stopAfterParseTests.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
```suggestion
#! /usr/bin/env bash
```
I think in some other scripts we use that way to point to bash, should we stick to that?